packages/frontend/src/components/profile.tsx

```diff
@@ -0,0 +1,11 @@
+interface ProfPic {
+  imgSrc: string;
+}
+
+export function Profile({ imgSrc }: ProfPic) {
+  return (
+    <div className={`w-[102px] h-[107px] rounded-2xl overflow-hidden`}>
```

Comment:
Bit of a minor nit pick, but could we use `w-24` and `h-24`. See [tailwindcss docs](https://tailwindcss.com/docs/width)

The benefit of this is that its using `rem` instead of `px` (meaning it will work better for multiple phone screen sizes) and also makes sure its square which I'm guessing was a mistake in the figma design. However this will be a bit bigger
```suggestion
    <div className={`w-24 h-24 rounded-2xl overflow-hidden`}>
```

Reply:
Should be done by now!